js/exporter/jspdf/v3/draw_utils.js

```diff
@@ -22,7 +22,7 @@ function drawCellText(doc, cell, docStyles) {
     if(isDefined(cell.text) && cell.text !== '') { // TODO: use cell.text.trim() ?
         const { textColor, font } = cell;
         setTextStyles(doc, { textColor, font }, docStyles);
-        drawTextInRect(doc, cell.text, cell._rect, cell.verticalAlign, cell.wordWrapEnabled, cell.jsPdfTextOptions);
+        drawTextInRect(doc, cell.text, cell.padding, cell._rect, cell.verticalAlign, cell.wordWrapEnabled, cell.jsPdfTextOptions);
```

Comment:
I think we can calculate text rect in this scope. And pass it to the drawTextInRect function.
Something like:

```js
    const textRect = {
        x: rect.x + padding.left,
        y: rect.y + padding.top,
        w: rect.w - (padding.left + padding.right),
        h: rect.h - (padding.top + padding.bottom)
    };
    
    
    drawTextInRect(doc, cell.text, cell.padding, textRect, cell.verticalAlign, cell.wordWrapEnabled, cell.jsPdfTextOptions);
```

Reply:
Done